Thinktecture.Relay.Server.Test/Diagnostics/TraceManagerTest.cs

```diff
@@ -109,7 +112,7 @@ public void Trace_creates_tracefiles_directory()
 				},
 				Body = new byte[] { 65, 66, 67 }
 			};
-			var onPremiseTargetResponse = new OnPremiseTargetResponse()
+			var onPremiseConnectorResponse = new OnPremiseConnectorResponse()
```

Comment:
In the previous version, a mocked class is used instead the real implementation. Shouldn't this be done again?

Reply:
Yes, done ;)